.github/workflows/scheduled_release.yml

```diff
@@ -0,0 +1,34 @@
+on:
+
+  schedule:
+    - cron: '0 0 * * 2'  # https://crontab.guru/#0_0_*_*_2
+
+name: Create Release
+
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Get current date
+        id: date
+        run: echo "name=$(date +'%Y-%m-%d')" >> $GITHUB_ENV
+      - name: Test with environment variables
+        run: echo $TAG_NAME - $RELEASE_NAME
+        env:
+          TAG_NAME: weekly-tag-${{ steps.date.outputs.date }}
+          RELEASE_NAME: weekly-release-${{ steps.date.outputs.date }}
```

Comment:
Can we maybe prefix the weekly/nightly builds with current version number as well? So instead of `weekly-release-2021-01-20`, we have `weekly-release-1.0.0-20210120`. Or better yet, `superset-1.0.0-20210120`.